src/profile-logic/import/chrome.js

```diff
@@ -376,8 +400,13 @@ async function processTracingEvents(
   const profile = getEmptyProfile();
   profile.meta.product = 'Chrome Trace';
 
-  let profileEvents: (ProfileChunkEvent | CpuProfileEvent)[] =
-    (eventsByName.get('ProfileChunk'): any) || [];
+  // Choose 500us as a somewhat reasonable sampling interval. When converting
```

Comment:
No action needed: I wonder if our tracing format is good enough now to get rid of this pseudo-sampling.

Reply:
Can you clarify what you mean by "our tracing format"?